core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -129,9 +129,13 @@ class RDDDataDistribution private[spark](
     val memoryUsed: Long,
     val memoryRemaining: Long,
     val diskUsed: Long,
+    @JsonDeserialize(contentAs = classOf[JLong])
```

Comment:
what does this mean?

Reply:
See jackson documentation. It tells jackson to deserialize the contents of a container as a specific type.